src/Infrastructure/UserRepository/UserRepo.ts

```diff
@@ -93,6 +95,10 @@ class UserRepo implements IUserRepo {
 
     return dbUser as IUserProps;
   }
+  async deleteAll() {
+    const count = await this.client.user.deleteMany();
```

Comment:
why do you need to store this? directly return it from this

Reply:
this is just for personal use to clean the database. would eliminate it in future